packages/mask/src/plugins/IdeaMarket/SNSAdaptor/AccountView.tsx

```diff
@@ -0,0 +1,136 @@
+
+const useStyles = makeStyles()((theme) => {
+    return {
+        empty: {
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            justifyContent: 'center',
+            height: '100%',
+            padding: theme.spacing(8, 0),
+        },
+        actionButtons: {
+            width: 96,
+        },
+        name: {
+            textOverflow: 'ellipsis',
+            overflow: 'hidden',
+        },
+        nameHeader: {
+            width: 200,
+        },
+        market: {
+            color: 'rgba(8,87,224,1)',
+            marginLeft: theme.spacing(0.25),
+        },
+        row: {
+            '&:nth-of-type(odd)': {
+                backgroundColor: theme.palette.background.default,
+            },
+        },
+    }
+})
+
+export function AccountView() {
+    const { t } = useI18N()
+    const { classes } = useStyles()
+    const account = useAccount()
+    const { value, error, loading } = useFetchUserTokens(account)
+    const userTokenBalances = value?.ideaTokenBalances
+
+    if (loading) {
+        return (
+            <div className={classes.empty}>
+                <LoadingAnimation />
+            </div>
+        )
+    }
+
+    if (error || !value) {
+        return (
+            <Typography className={classes.empty} color="textPrimary">
+                {t('plugin_ideamarket_smthg_wrong')}
+            </Typography>
+        )
+    }
+
+    return (
+        <>
+            <TableContainer component={Paper} sx={{ width: '100%' }}>
+                <Table sx={{ tableLayout: 'fixed', width: '100%' }} size="small" aria-label="simple table">
+                    <TableHead>
+                        <TableRow>
+                            <TableCell className={classes.nameHeader} key="name">
+                                Idea
+                            </TableCell>
+                            <TableCell key="price">Price</TableCell>
+                            <TableCell key="balance">Balance</TableCell>
+                            <TableCell key="value">Value</TableCell>
+                        </TableRow>
+                    </TableHead>
+                    <TableBody>
+                        {userTokenBalances.map((balance: UserIdeaTokenBalance) => {
+                            const name =
+                                balance.token.name.length > 30
+                                    ? balance.token.name.slice(0, 30).concat('...')
+                                    : balance.token.name
+                            const tokenPrice = Number.parseFloat(
+                                balance.token.latestPricePoint.price.toString(),
+                            ).toFixed(2)
+                            const userTokenBalance = leftShift(balance.amount, 18).toFixed(2, 1)
+                            const balanceValue =
+                                balance.token.latestPricePoint.price * leftShift(balance.amount, 18).toNumber()
```

Comment:
Recommend to use `scale10`.

Reply:
can you get more precise please ?

Reply:
`scale10(balance.token.latestPricePoint.price, 18)`

Reply:
`balance.token.latestPricePoint.price` is already in decimals. I don't understand the point of scale10 here.